src/ui/flutter_app/lib/l10n/intl_en.arb

```diff
@@ -272,25 +292,55 @@
     "@black": {
         "description": "Player 2"
     },
-    "loseReasonlessThanThree": " piece count is less than three.",
+    "loseReasonlessThanThree": "{loser} piece count is less than three.",
     "@loseReasonlessThanThree": {
-        "description": " piece count is less than three."
-    },
-    "loseReasonResign": " resign.",
+        "description": "Shows that the player lost because they had less than three pieces.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonResign": "{loser} resign.",
     "@loseReasonResign": {
-        "description": " resign."
-    },
-    "loseReasonNoWay": " is no way to go.",
+        "description": "Shows that the player lost because they had resigned.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonNoWay": "{loser} is no way to go.",
     "@loseReasonNoWay": {
-        "description": " is no way to go."
-    },
-    "loseReasonBoardIsFull": "The board is full, and no way to go.",
+        "description": "Shows that the player lost because they had no way to go.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonBoardIsFull": "{loser} The board is full, and no way to go.",
```

Comment:
This is not needed.

Reply:
What do you mean by that?

Reply:
The word order is wrong, right? Or we can amend to: "The board is full, and {loser} no way to go."

Reply:
yeah I just used the old representation but I'll change it for the new strings